src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/customer_preferences.yml

```diff
@@ -13,3 +13,21 @@ admin_customer_preferences_process:
     _controller: 'PrestaShopBundle\Controller\Admin\Configure\ShopParameters\CustomerPreferencesController::processAction'
     _legacy_controller: AdminCustomerPreferences
     _legacy_link: AdminCustomerPreferences:update
+
+admin_customer_groups_index:
+  path: /groups
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Configure\ShopParameters\CustomerGroupsController::indexAction'
+    _legacy_controller: AdminGroups
+    _legacy_link: AdminGroups
+
+admin_customer_groups_search:
+  path: /groups
+  methods: POST
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
+    gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.customer_groups
+    redirectRoute: admin_customer_groups_index
+    _legacy_controller: AdminGroups
+    # _legacy_link: AdminGenders:submitFiltergenders
```

Comment:
These routes are duplicate with the ones you placed in the `customer_groups.yml` file, it's not necessarily a problem to add them there, since they belong in the same section (Tab) as the customer preferences it makes sense to decide to group them

Or we can choose to keep them separated for cleaner/small files, but we can't do both So choose either one but don't duplicate routes please

Reply:
I just forgot to remove them here